bgzf/bgzf_test.go

```diff
@@ -1034,3 +1037,37 @@ func BenchmarkWrite(b *testing.B) {
 		bg.Wait()
 	}
 }
+
+func BenchmarkRead(b *testing.B) {
+	if *file == "" {
+		b.Skip("no bgzf file specified")
+	}
+	fh, err := os.Open(*file)
+	if err != nil {
+		b.Fatalf("file open failed: %v", err)
+	}
+	defer fh.Close()
+
+	buf := make([]byte, 16384)
+	b.ResetTimer()
+	//b.ReportAllocs()
+
+	for i := 0; i < b.N; i++ {
+		fh.Seek(0, os.SEEK_SET)
+		bgz, err := NewReader(fh, *conc)
+		if err != nil {
+			b.Fatalf("bgzf open failed: %v", err)
+		}
+		for {
+			_, err = bgz.Read(buf)
+			if err == io.EOF {
+				break
+			}
+			if err != nil {
+				b.Fatalf("bgzf read failed: %v", err)
+			}
+		}
+		bgz.Close()
+	}
+	b.ReportAllocs()
```

Comment:
Please remove this; you can get allocation reporting during a benchmark by `-test.benchmem`.